test/test_vector.py

```diff
@@ -771,6 +771,25 @@ def test_copy_overload():
     assert vectorequal(b, original)
 
 
+def test_vector_values():
+    x = Array(values=[1.0, 2.0, 3.0, 4.0, 5.0], unit="m")
+    y = Array(values=[6.0, 7.0, 8.0, 9.0, 10.0], unit="m")
+    z = Array(values=[11.0, 12.0, 13.0, 14.0, 15.0], unit="m")
+    v = Vector(x, y, z)
+    val = v.values
+    expected = np.array(
+        [
+            [1.0, 6.0, 11.0],
+            [2.0, 7.0, 12.0],
+            [3.0, 8.0, 13.0],
+            [4.0, 9.0, 14.0],
+            [5.0, 10.0, 15.0],
+        ]
+    )
+    assert val.shape == (5, 3)
+    assert np.all(val == expected)
+
```

Comment:
```suggestion
    assert np.array_equal(val[:, 0], x.values)
    assert np.array_equal(val[:, 1], y.values)
    assert np.array_equal(val[:, 2], z.values)
```

Reply:
This is to make sure that we did not get the order wrong in the test when writing `expected` ;-)